integration-tx-api/src/main/java/org/apache/seata/integration/tx/api/interceptor/handler/AbstractProxyInvocationHandler.java

```diff
@@ -28,7 +28,7 @@ public abstract class AbstractProxyInvocationHandler implements ProxyInvocationH
 
     protected abstract Object doInvoke(InvocationWrapper invocation) throws Throwable;
 
-    protected int order = Integer.MAX_VALUE;
+    protected int order = 0;
```

Comment:
```suggestion
    protected int order =Integer.MAX_VALUE;
```
I think we should continue to use Integer.MAX_VALUE for this value, and then return 0 in the order method of GlobalTransactionalInterceptorHandler, as the order for GlobalTransactionalInterceptorHandler.

Reply:
Considering the existing handler order, this seems like a good approach.